applications/nova-budapest.md

```diff
@@ -0,0 +1,258 @@
+- Dev 2: smart contract creator
+- Dev 3: marketplace (Opensea, etc.) developer + minting code developer 
+- Writer / proofreader
+- Webdesigner
+
+### Ecosystem Fit
+
+We think as more and more game companies are experimenting with blockchain additions to their projects, or even introducing tokes as in-game currency, we can declare, that a computer game is definitely a fit to a blockchain ecosystem.
+
+As mentioned above, we target both IRL lifestyle subvese inhabitants, RP-ers and classic players. We are convinced, that our project art and direction will resonate with the "classic" and "neo-classic" anime and MMORPG lovers, and they will find this ever changing world immersive enough to play.
+
+We have no knowledge of a project with this details and uniqueness (nor, a game project as a matter of fact) in the Polkadot ecosystem. We know only one project called [Big Time](https://bigtime.gg/) in the Ethereum ecosystem.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+Submitted in Google Forms in detail with LinkedIn profiles and relevant experiences.
+
+### Contact
+
+- **Contact Name:** Submitted in Google Forms
+- **Contact Email:** Submitted in Google Forms
+- **Website:** Submitted in Google Forms
+
+### Legal Structure
+
+- **Registered Address:** Submitted in Google Forms
+- **Registered Legal Entity:** Submitted in Google Forms
+
+### Team's experience
+
+Submitted in Google Forms in detail with LinkedIn profiles and relevant experiences.
+
+### Team Code Repos
+
+Submitted in Google Forms.
+
+### Team LinkedIn Profiles (if available)
+
+Submitted in Google Forms.
+
+## Development Status :open_book:
+
+Ongoing. We already submitted the relevant repositories and ready to present the art. No 3D art is ready at the moment.
+
+## Development Roadmap :nut_and_bolt:
+
+### Laying Down the Foundations (Done)
+
+### Creating key and character arts (60% readiness)
+In this step the key arts for the characters will be created as listed in this paragraph. These arts cannot be cheap, and should reflect the highest quality available.
+
+Key arts are one of the main pillars of a successful fundraising as these characters will be minted into NFT’s, and will be the 3D characters in the game. 
+
+### Establishing basic infrastructure (20% readiness)
+In this phase the developers create the project website, decide on the blockchain and NFT technology. We train an AI assistant with the game lore to help newcomers. After they set up the smart contract and the minting and web accounts.
+
+### Pre-development
+The game won’t be able to penetrate to wide audience without any basic gameplay. This needs to be created with 2-3 characters modeled and animated, and some environment pre-modelled, even 4-5 quests playable. Optimally vehicles and VR e-learning demo. Optimal length is 45 seconds of gameplay video.
+
+### Overview
+
+- **Total Estimated Duration:** 8 months
+- **Full-Time Equivalent (FTE):** 5 FTE
+- **Total Costs:** 104,000 EUR
```

Comment:
We usually expect new teams to apply for a L1 or L2 grant, rather than a L3.

Reply:
Understood, will refactor related to this.